### ISSUE TITLE
TOML configuration

### ISSUE THREAD

**Issue by svanburen** [uber/prototool#395]
Hi,

I'd like for prototool to support the TOML language for configuration. I feel as though YAML is a bit ambiguous from a spec perspective, and JSON is not intended to be a configuration language.

There are some reasonable TOML parsers in Go which would probably make this trivial: https://github.com/BurntSushi/toml, https://github.com/pelletier/go-toml.

Would be happy to put together a PR for such support. Any thoughts?

**Comment by peter-edge** [comment]
I'm not crazy about the idea - we only added JSON half-heartedly, and most tools don't support multiple formats. For example, Travis CI doesn't support .travis.json or .travis.toml as far as I'm aware.

I feel like adding another supported configuration method leaves more room for documentation and usage issues. Prototool is heavy on having one way to do things, and honestly I wish we didn't support JSON files.

What is ambiguous from a spec perspective? Can you give me specific examples?

**Comment by peter-edge** [comment]
I haven't heard the case here - I'm going to close this for now. Unless there's a compelling case for why we should handle `prototool.toml`, I rather there be less ways to do things so there's less confusion for users.

**Comment by svanburen** [comment]
Fair enough, and sorry for the delayed response. I still personally find that TOML works better as a configuration language than YAML, but I acknowledge that the tool already has a standard configuration that's straightforward enough. Cheers!
